waku/waku_rln_relay/rln/wrappers.nim

```diff
@@ -89,7 +89,7 @@ proc createRLNInstanceLocal(d = MerkleTreeDepth,
       cache_capacity: 15_000,
       mode: "high_throughput",
       compression: false,
-      flush_interval: 12_000,
+      flush_interval: 500,
```

Comment:
What is the flush_interval attribute used for?

Reply:
This is describes how often we flush pmtree cache to disk. just experimenting with waku-simulator here